apollo-gradle-plugin/src/main/kotlin/com/apollographql/apollo/gradle/internal/DefaultCompilationUnit.kt

```diff
@@ -99,11 +99,13 @@ abstract class DefaultCompilationUnit @Inject constructor(
       }
     }
 
-    private fun multipleSchemaError(schemaList: List<File>): String {
+    private fun multipleSchemaError(project: Project, schemaList: List<File>): String {
       val services = schemaList.joinToString("\n") {
+        val name = it.parentFile.name
```

Comment:
Are you sure the parentFile is enough? What happens if there are multiple nested folders? It might be better to `relativize` to the project folder?

Reply:
This is mostly informative and I wanted to keep it simple for most users where I expect `parentFile.name` to be different accross services. Another option would just be to use placeholders?

```
apollo {
  service("service1") {
    ...
  }
  service("service2") {
    ...
  }
}
```

Reply:
When you first did these warnings, they were really close to be something that the user can copy paste and fix their issue. Although I like that idea, it is also too much maintainance to keep that.

That's why maybe placeholders is a good idea. Another idea is to also link to the documentation??

Reply:
Link to the documentation is a great idea 👍 . I don't think we have a page talking about multiple services yet. Let me see if I can add one quickly

Reply:
👍Just the page talking about service would be enough actually. Up to you.

Reply:
Ah yes, this looks enough: https://www.apollographql.com/docs/android/essentials/plugin-configuration/#apolloextension-services-and-compilationunit 👍